app/features/feedback/shared/components/text-input.jsx

```diff
@@ -1,16 +1,13 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import uuidv4 from 'uuid/v4';
 
 function TextInput({ title, help, onChange, name, type = 'text', required = false, value }) {
-  const uuid = uuidv4();
   return (
     <fieldset>
-      <label htmlFor={uuid}>
+      <label>
```

Comment:
The input isn't nested in the label here, so wouldn't you need for/id in this case?